code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -244,11 +255,6 @@ func (sc *StorageSmartContract) newAllocationRequestInternal(
 			"Not enough provided blobbers found in mpt")
 	}
 
-	if request.Owner == "" {
-		request.Owner = txn.ClientID
```

Comment:
won't it break anything?

Reply:
we are already making the same check a few lines above
https://github.com/0chain/0chain/blob/d4c3475e2f32f835219d24e80d747c117c33d7d9/code/go/0chain.net/smartcontract/storagesc/allocation.go#L238-L241